ascent/src/aggregators.rs

```diff
@@ -16,20 +16,29 @@ use std::iter::Sum;
 use std::ops::Add;
 
 /// computes the minimum of the input column
-pub fn min<'a, N: 'a>(inp: impl Iterator<Item = (&'a N,)>) -> impl Iterator<Item = N>
-where N: Ord + Clone {
+pub fn min<'a, N>(inp: impl Iterator<Item = (&'a N,)>) -> impl Iterator<Item = N>
+where
+   N: Ord + Clone,
```

Comment:
Could you collapse these into one line? The same goes for other places.

Reply:
Not sure what you mean by this?

This is what `cargo fmt` (as per https://github.com/s-arash/ascent/issues/24#issuecomment-1986089882) produces.

Reply:
I meant doing `where N: 'a + Ord + Clone`, assuming `cargo fmt ` finds that acceptable!

Reply:
The lines get split like that by `cargo fmt`, based on the where clause's length, etc. So no.

Reply:
I just tried `cargo +nightly fmt` with `where N: 'a + Ord + Clone`, and it didn't break it down into two lines 🤔.

Reply:
Oh, sorry I missed the change from `N: Ord + Clone, N: 'a` to `N: 'a + Ord + Clone`.
The latter does indeed not get split up. Fixed.

Reply:
Nice. Thanks.